app/coffee/Features/Collaborators/CollaboratorsHandler.coffee

```diff
@@ -1,13 +1,73 @@
 UserCreator = require('../User/UserCreator')
 Project = require("../../models/Project").Project
-ProjectEntityHandler = require("../Project/ProjectEntityHandler")
 mimelib = require("mimelib")
 logger = require('logger-sharelatex')
 UserGetter = require "../User/UserGetter"
 ContactManager = require "../Contacts/ContactManager"
 CollaboratorsEmailHandler = require "./CollaboratorsEmailHandler"
+async = require "async"
 
 module.exports = CollaboratorsHandler =
+	getMemberIdsWithPrivilegeLevels: (project_id, callback = (error, members) ->) ->
+		Project.findOne { _id: project_id }, { owner_ref: 1, collaberator_refs: 1, readOnly_refs: 1 }, (error, project) ->
+			return callback(error) if error?
+			return callback null, null if !project?
+			members = []
+			members.push { id: project.owner_ref.toString(), privilegeLevel: "owner" }
```

Comment:
we pass strings around alot and check them. Worth thinking about having a file with object we use (enum's)

if privlageLevel == privlageLevels.owner
.....